documentation/src/main/asciidoc/topics/config_examples/jgroups_relay.xml

```diff
@@ -0,0 +1,17 @@
+<jgroups>
+   <!-- Uses the default JGroups UDP stack for intra-cluster traffic. -->
+   <stack name="xsite" extends="udp">
+      <!-- Adds RELAY2 to the stack for inter-cluster transport. -->
+      <!-- Names the local site. Data in caches from the local site is replicated to backup locations. -->
+      <!-- Configures a maximum of 1000 site masters for the local cluster. -->
+      <relay.RELAY2 xmlns="urn:org:jgroups"
+                    site="LON"
+                    max_site_masters="1000"/>
+      <!-- Specifies all site names and uses the default JGroups TCP stack for inter-cluster transport. -->
+      <remote-sites default-stack="tcp">
+         <!-- Names each remote site as a backup location. -->
```

Comment:
this is actually "name all sites" :)

Reply:
thanks @pruivo changing to "Names all sites that participate in cross-site replication."